src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,128 @@
+import React, {
+  FC,
+  CSSProperties,
+  useState,
+  Children,
+  ReactElement,
+  cloneElement,
+  ReactNode,
+} from 'react';
+import { classNames } from '../../utils/classNames';
+import DropdownOption from './DropdownOption';
+
+export interface IDropdownProps {
+  /** children must be React Element */
+  children?: ReactElement | ReactElement[];
+  /** set customized css class */
+  className?: string;
+  /** set dropDown to be disabled */
+  disabled?: boolean;
+  /** set customized css style */
+  cssStyle?: CSSProperties;
```

Comment:
why you rename it as cssStyle? style is a HTML attribute. if you are not going to change the default behavior of it. you could just use it